iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/persistence/pipe/PipeTaskInfo.java

```diff
@@ -80,12 +84,51 @@ private void releaseReadLock() {
 
   private void acquireWriteLock() {
     pipeMetaKeeper.acquireWriteLock();
+    pipeTaskInfoVersion.increaseLatestVersion();
   }
 
   private void releaseWriteLock() {
     pipeMetaKeeper.releaseWriteLock();
   }
 
+  /////////////////////////////// Version ///////////////////////////////
+
+  private class PipeTaskInfoVersion {
+
+    private final AtomicLong latestVersion;
+    private long lastSyncedVersion;
+    private boolean isLastSyncedPipeTaskInfoEmpty;
+
+    public PipeTaskInfoVersion() {
+      this.latestVersion = new AtomicLong(0);
+      this.lastSyncedVersion = 0;
+      this.isLastSyncedPipeTaskInfoEmpty = false;
+    }
+
+    public void increaseLatestVersion() {
+      latestVersion.incrementAndGet();
+    }
+
+    public void updateLastSyncedVersion() {
+      lastSyncedVersion = latestVersion.get();
+      isLastSyncedPipeTaskInfoEmpty = pipeMetaKeeper.isEmpty();
+    }
+
+    public boolean isEmptySynced() {
```

Comment:
isEmptySynced infers not only the very last empty sync, but the empty pipeMeta now. Maybe simply call it "canSkipNextSync"?